test_apps/coverage_app/app/components/blockchain.js

```diff
@@ -0,0 +1,91 @@
+import EmbarkJS from 'Embark/EmbarkJS';
```

Comment:
Do you need everything that is present in the demo in terms of client side code (React code)?
It means we would have to apply our changes twice.

I feel you only need contracts element?

Reply:
You're right. I only need the contracts element.